modules/lymphgen/1.0/config/default.yaml

```diff
@@ -65,9 +62,10 @@ lcr-modules:
                 bcl6falsevalues: "__UPDATE__"
             lymphgen_input:
                 seq_type: "__UPDATE__"  # Availible options: exome/genome (they do the same thing) and targeted (for panels)
-                use_log_ratio: "__UPDATE__"
+                use_log_ratio: "True"
             lymphgen_run:
-                lymphgen_path: "/projects/rmorin/software/lcr-scripts/LymphGen/Run_Lymphgen.R"
+                lymphgen_path: "/home/fymoayyed/LymphGen/Run_Lymphgen.R"
```

Comment:
I think it's actually better to make this 
```
lymphgen_path: "__UPDATE__"
```

We have had some external users make use of this module to prepare their files for the LymphGen webtool, so ideally the default is that users have to specify where they've saved the source code if they even have access to it.

Reply:
Just realized I contradicted my earlier comment about this, hah. I think it's better to leave this as "__UPDATE__" still!